### DISCUSSION TITLE
Access mesh rotation as done with mesh.center ?

### DISCUSSION THREAD

**Discussion by leo2r** [pyvista/pyvista#3450] [Q&A] [answered]
Hi,

Is there an easy way to access the current rotation state (change in rotation from originally imported mesh state), in a similar way to access the centre of the mesh with mesh.center? Or do I have to keep track of it manually through a delta rotation variable?

Thanks :)

**Comment by banesullivan** [comment, marked as answer]
You'll need to keep track of this manually. That information is not (currently) captured during any of the `rotation_*` filters

**Reply by leo2r** [reply]
Ok thanks! :)
